### ISSUE TITLE
not support 16.04

### ISSUE THREAD

**Issue by wesky93** [U-lis/initial-settings#1]
je gut ne

**Comment by U-lis** [comment]
@wesky93 ubuntu 16.04 uses gnome 3.18 as default.
And in gnome 3.8+, config command is changed from gconftool-2 to dconf.
I fixed and tested on my VM ubuntu 16.04 so please test yourself and give me some feedbacks.
Thx.

**Comment by wesky93** [comment]
I checked my notebook ubnut16.04.. but it doesn't work.. will you check one more time?
shell script says... 

```
Inconsolata-dz for powerline Installed
DejaVu Sans Mono for powerline Installed
You choose theme agnoster so we will set font to Inconsolata-dz for Powerline
./font/font-settings.sh: 줄 18: gnome-shell: 명령어를 찾을 수 없음
./font/font-settings.sh: 줄 22: [: : integer expression expected
Terminal font change done
```
